src/Mod/Draft/draftgeoutils/faces.py

```diff
@@ -185,14 +192,25 @@ def create_face(w1, w2):
             faces_fused_list = []
             for faces in faces_list:
                 if len(faces) > 1 :
-                    faces_fused = faces[0].fuse(faces[1:]).removeSplitter().Faces[0]
-                    faces_fused_list.append(faces_fused)
+                    # Below not good if a face is self-intersecting or reversed
```

Comment:
Is reversed is really an issue here?

Reply:
Self-intersecting and 'reversed' create problem as discussed in the FC forum and here, though seems the code have not correctly identify what is 'reversed' in the current problem.